shlr/ar/Makefile

```diff
@@ -26,7 +26,7 @@ endif
 endif
 endif
 
-CFLAGS+=-I../../libr/include
+CFLAGS+=-I../../libr/include -I../sdb/src/
```

Comment:
ideally i would prefer to include sdb/deps.mk insteada of having this everywhere duplicated